scripts/ci/run-vagrant-tests.sh

```diff
@@ -12,51 +12,116 @@ set -v
 
 dir="$(dirname "$0")"
 
-cd ${GOPATH}/src/github.com/skydive-project/skydive
+root=${GOPATH}/src/github.com/skydive-project/skydive
+cd $root
+
+if [ "$DEVMODE" = "true" ]; then
+    make
 
-if [ "$DEVMODE" == "true" ]; then
-    make static
     make srpm
     VERSION=$(make -s version | cut -d '-' -f 1)
     TAG=$(make -s version | cut -s -d '-' -f 2- | tr '-' '.')
     mock -r centos-7-x86_64 -D "fullver $(make -s version)" --resultdir . --rebuild rpmbuild/SRPMS/skydive-${VERSION}*${TAG}.src.rpm
     mkdir -p rpmbuild/RPMS/{x86_64,noarch}
     mv skydive*-${VERSION}-*${TAG}*.x86_64.rpm rpmbuild/RPMS/x86_64/
     mv skydive*-${VERSION}-*${TAG}*.noarch.rpm rpmbuild/RPMS/noarch/
+
     make docker-image
     docker save skydive/skydive:devel -o skydive-docker-devel.tar
+
+    make test.functionals.compile
 fi
 
 cd contrib/vagrant
 
+export ANALYZER_COUNT=1
```

Comment:
A lof of effort was put into making the etcd clustering setup by ansible work properly, and this job is the only way we test it (with 2 analyzers). Is there any reason to use only analyzer ?

Reply:
For stability reason, I will switch back to 2 analyzers, 2 agents in another patchset

Reply:
when I started 2 analyzers wasn't working at all, so I'm sure we still have bug with